client/components/dashboard/MealHistory.jsx

```diff
@@ -1,23 +1,162 @@
 import PropTypes from 'prop-types';
 import './MealHistory.css';
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
 
 export default function MealHistory({ meals }) {
+  const [mealHistory, setMealHistory] = useState(null);
```

Comment:
might be better to use empty array similar to `client/components/dashboard/WorkoutHistory.jsx`
```suggestion
  const [mealHistory, setMealHistory] = useState([]);
```

Reply:
Thanks